ledger/error/src/main/scala/com/daml/error/ErrorCode.scala

```diff
@@ -275,9 +275,8 @@ object ErrorCode {
 }
 
 // Use these annotations to add more information to the documentation for an error on the website
-case class Deprecation(deprecation: String) extends StaticAnnotation
 case class Explanation(explanation: String) extends StaticAnnotation
 case class Resolution(resolution: String) extends StaticAnnotation
 case class Description(description: String) extends StaticAnnotation
 case class RetryStrategy(retryStrategy: String) extends StaticAnnotation
-case class DeprecatedDocs(description: String) extends StaticAnnotation
+case class DeprecatedDocs(deprecation: String) extends StaticAnnotation
```

Comment:
`Deprecation` and `DeprecatedDocs` seemed to be redundant. Arbitrarily chose to use `DeprecatedDocs` and remove the other

Reply:
Yes, they are redundant. This quirk stemmed from the impossibility of extracting the message from the Scala `@deprecated` annotation for both 2.12 and 2.13 using the same code ❗ Now that we've dropped 2.12, I would even revert to Fabio's change: Remove `DeprecatedDocs` and just use `deprecated`